photon-ml/src/main/scala/com/linkedin/photon/ml/cli/game/training/Driver.scala

```diff
@@ -514,18 +538,22 @@ final class Driver(val params: Params, val sparkContext: SparkContext, val logge
 
     trainingDataSet.foreach { case (_, rddLike: RDDLike) => rddLike.unpersistRDD() }
 
+    timer.start()
+    val bestModel = selectBestModel(gameModelsMap, validatingDataAndEvaluatorsOption)
+    timer.stop()
+    logger.info(s"Time spent in model evaluation: ${timer.durationSeconds} (s)\n")
```

Comment:
Doesn't fit in with the other log statements. Should say something like:
```
Time elapsed after selecting best model:
```

Also, this block should be within the `if (modelOutputMode != ...)` statement.